BuildResidentialHPXML/measure.rb

```diff
@@ -1064,6 +1070,18 @@ def arguments(model)
     arg.setDefaultValue(false)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('cooling_system_airflow_defect_ratio', false)
+    arg.setDisplayName('Cooling System: Airflow Defect Ratio')
+    arg.setDescription("The airflow defect ratio, defined as (InstalledAirflow - DesignAirflow) / DesignAirflow, of the cooling system. A value of zero means no airflow defect. Applies only to #{HPXML::HVACTypeCentralAirConditioner}.")
```

Comment:
Also applies to a _ducted_ mini-split AC. (It does not apply to a ductless mini-split.)